docbook/Admin_Guide/en-US/config/auth.xml

```diff
@@ -171,7 +200,7 @@ ldaps://ldap.example.com:3269/
 					automatically follows referrals returned by LDAP
 					servers or not. This maps to LDAP_OPT_REFERRALS ldap
 					library option. Defaults to
-					<emphasis>ON</emphasis>.</para>
+					<emphasis>OFF</emphasis>.</para>
```

Comment:
The final decision was to keep it as it was before (`ON`)

Reply:
Yeah, I'll leave it alone (still `ON`). I added a comment saying it might be best to set to `OFF`.